components/script/dom/webidls/HTMLFormElement.webidl

```diff
@@ -17,7 +17,7 @@ interface HTMLFormElement : HTMLElement {
 
   [SameObject] readonly attribute HTMLFormControlsCollection elements;
   readonly attribute unsigned long length;
-  //getter Element (unsigned long index);
+  getter Element? (unsigned long index);
```

Comment:
This isn't following the [WebIDL definitions for HTMLFormElement](https://html.spec.whatwg.org/multipage/forms.html#htmlformelement).